test/Sema/implementation-only-import-in-decls.swift

```diff
@@ -87,18 +87,26 @@ public var testMultipleBindings1: Int? = nil, testMultipleBindings2: BadStruct?
 public var testMultipleBindings3: BadStruct? = nil, testMultipleBindings4: Int? = nil // expected-error {{cannot use struct 'BadStruct' here; 'BADLibrary' has been imported as implementation-only}}
 
 extension BadStruct { // expected-error {{cannot use struct 'BadStruct' here; 'BADLibrary' has been imported as implementation-only}}
```

Comment:
Can the message at least say something like "Cannot use struct 'BadStruct' in extension with public members"? Or use a note or something, I don't care—just let the user know somehow that the problem is that the extension has public members, because if you don't say so, they'll have no idea what's wrong.